invoke/parser/parser.py

```diff
@@ -88,7 +88,7 @@ def parse_argv(self, argv):
             # flag value and still in valid parsing territory (i.e. not in
             # "unknown" state which implies store-only)
             if not machine.waiting_for_flag_value and is_flag(token) \
-                and not machine.result.unparsed:
+                    and not machine.result.unparsed:
```

Comment:
Is 8-space indents on continuations a flake8 thing? It's not my preferred style :( (Honestly, I thought flake8 would complain about backslash continuations entirely, parentheticals are better anyway).